articles/tech-lead-needed.md

```diff
@@ -0,0 +1,98 @@
+
+The question, hence, is this: How can the tech lead enable the team, id est, make them more productive? This is important stuff, so allow me to elaborate.
+
+### Developer Productivity
+
+For the sake of brevity, let us simplify things an enormous bit by saying this:
+
+* The project manager and the tech lead define the development process, documentation guidelines, and other general constraints.
+
+* The tech lead takes in the product vision, plus high-level requirements and cross-cutting concerns, and outputs a definition of the big picture, namely how the system is decomposed into components, what responsibilities these components have, and how these components are supposed to collaborate.
+
+* Each developer is responsible for a couple of components. She or he takes in requirements that affect these components, plus architectural decisions, and outputs implementations of the respective features that adhere to the architecture as well as all the other constraints.
+
+This may be a crude simplification (for example, many teams are mature enough to define proper guidelines on their own), but it suffices to understand a crucial point: The productivity of developers depends significantly on the quality of their input, that is, on the quality of requirements, the quality of architectural decisions, and the quality of the general constraints such as process definitions and documentation guidelines. Therefore, a tech lead can make one hell of a difference by making sure that the quality of this input is high.
+
+### Garbage In, Gold out?
+
+The last sentence can be put as a rule, too: The tech lead needs to act when the team is continuously supposed to produce high-quality output from low-quality input. Please note that I use the term *input* in a very broad sense here, namely in a sense that entails aspects such as processes, guidelines, and even the general conditions under which the team is supposed to work.
+
+Here are three corollaries to this rule:
+
+* The tech lead needs to team up with the project manager in order to help them define processes and guidelines that have a good balance between formal needs and everyday applicability.
```

Comment:
I see how you build up the article, but this sentence is basically the same information as above in: "The project manager and the tech lead define the development process, documentation guidelines, and other general constraints."
Unfortunately I see no better option option here